src/sql/media/icons/common-icons.css

```diff
@@ -528,6 +528,20 @@ Includes non-masked style declarations. */
 	background-image: url("database_colored.svg");
 }
 
+.notebookExplorer {
```

Comment:
Please use a better name - this isn't really descriptive of what the icon actually is. Common icons should be a place to put icons that can be reused by anything and so tying it to notebookExplorer isn't idea.

I'd suggest just using codicon.book to follow the convention of other icons in this file

Reply:
okay I have renamed it as book and moved the explorer specific code to a separate css.